kbfsedits/tlf_history.go

```diff
@@ -281,6 +283,23 @@ func (r *recomputer) processNotification(
 			return false
 		}
 
+		// See if any of the parent directories were renamed, checking
+		// backwards until we get to the TLF name.
+		prefix := filename
+		suffix := ""
+		for strings.Count(prefix, "/") > 4 {
```

Comment:
What about Windows?

Reply:
> What about Windows?

The path in the file edit notifications come from `libkbfs.Path`, which explicitly uses `/` to construct them: https://github.com/keybase/kbfs/blob/ae58f965dbcd49300d922dd09d0fff829b916de7/libkbfs/path.go#L29-L48

Any translation to system-specific paths should happen in the GUI.